### PR TITLE
update timestamp() function to something MySQL|MariaDB would accept

### PR DESCRIPTION
**Pull request by Robert-Krajcik**

### CODE REVIEW

**Comment by Robert-Krajcik** [comment]
Hi @AARON-CLARK 
Can you review this?  It's a pretty simple update, just used in the comments table.
Thanks.

**Comment by AARON-CLARK** [comment]
> Hi @AARON-CLARK
> Can you review this? It's a pretty simple update, just used in the comments table.
> Thanks.

@marlycormar and I were going to split up the work, but I don't mind reviewing this one too. It's pretty small. Stay tuned...

File: Utils/utils.R [hunk 1]
```diff
@@ -57,9 +57,10 @@ db_ins <- function(query){
 }
 
 TimeStamp <- function(){
-  Timestamp_intial<-str_replace(Sys.time()," ", "; ")
-  Timestamp <- paste(Timestamp_intial, Sys.timezone())
-  return(Timestamp)
+  # Timestamp_intial<-str_replace(Sys.time()," ", "; ")
+  # Timestamp <- paste(Timestamp_intial, Sys.timezone())
+  # return(Timestamp)
```
Comment (AARON-CLARK) [inline Utils/utils.R:62]: Suggest removing old code to keep things neat and clean

**Review by AARON-CLARK — CHANGES_REQUESTED** [review]
Just one small ask!

**Comment by Robert-Krajcik** [comment]
All set.

**Review by AARON-CLARK — APPROVED** [review]
lgtm

**Pull request merged**